src/pages/index.js

```diff
@@ -23,8 +38,23 @@ const cardSection = new Section(
   },
   "#card-add-form"
 );
-cardSection.renderItems();
 
+//Getting the list of cards from the server and displaying it on the form
+api
+  .getInitialCards()
+  .then((result) => {
+    return result;
+  })
+  .then((res) => {
+    res.forEach((result) => {
+      renderCard(result);
```

Comment:
Please, use `renderItems` for rendering the array of cards


Pass `items`  into the call for that

```js
renderItems(items) {
```